src/test/controllers/survey.controller.test.js

```diff
@@ -0,0 +1,10 @@
+// import '../../server/models/answer.model';
```

Comment:
Don't include this in this PR if you're going to leave them commented out. Or just complete the tests :)